src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Keyword-Strings.json

```diff
@@ -118,6 +118,10 @@
     "NAMES": ["patience"],
     "DESCRIPTION": "Gain 1 #yCoil each time you play a card. When you remember a new memory, deal 1 damage to ALL enemies for each #yCoil and lose all #yCoil."
   },
+  {
+    "NAMES": ["possessed"],
+    "DESCRIPTION": "#yPossessed creatures will deal damage equal to it's max HP to a random ally for each #yPossessed stack."
```

Comment:
"its"

Reply:
fixed. Also adjusted "max HP" to be consistent across all of our cards